### DISCUSSION TITLE
Moving from print to digital

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5921] [Accessibility]
Digital content provides folks with a lot of flexibility. They can customize their display or translate content into other languages. But a lot of important government information is still available only on paper or in documents people have to download, like PDFs. Users may face barriers when they can’t customize or translate these documents.

What can teams do to make sure important information is available in an accessible digital format? How can the design system help with this transition?

**Comment by sarah-sch** [comment]
A good first step might be an audit of all of your forms. Which ones are only available in print or as PDFs/downloads? Which ones are used the most and might therefore be good candidates to prioritize for digitalization? I'm curious about the experience of teams who have started or completed this kind of digitization process. What was your experience like?

**Reply by melissa-wa** [reply]
One barrier I've heard in my state agency is the infrequent-but-necessary ability to print a blank form for a customer (e.g., someone without internet access or comfort navigating online) or to print a completed form for record-keeping purposes (e.g., HR may need to redact confidential info on a form, then save to a personnel file). To prevent duplicating work, we're considering how we can make web forms that can be printed/saved to PDF in accessible format directly from websites and from within our intake/collection software.

I'd also be interested to hear about experiences with the process of digitizing forms in gov. What tools have folks used?

**Reply by sarah-sch** [reply]
@melissa-wa - You make a great point that forms _do_ need to be printed sometimes, so it's not about moving from printable to completely digital but rather making forms available for everyone in the format that works for the situation. In addition to the administrative situations you reference, I suspect there are some users who are just more comfortable with a hard copy of a form than they are with a digital one ... or more comfortable getting help in person or over the phone than they are getting guidance online in the first place.

**Reply by melissa-wa** [reply]
Exactly, @sarah-sch - meeting folks where they're at is so important. Btw your suggestion about PDF inventory/audit has inspired me to make that a priority. We already have our # 1 priority for digitization identified, but I think a full audit will help us assess where we're at and how to move forward. So, thank you for the suggestion!

**Reply by sarah-sch** [reply]
That's awesome--thanks for sharing, @melissa-wa!
